accessibilityTest/AccessibilityQuickview.ts

```diff
@@ -1,33 +1,101 @@
 import * as axe from 'axe-core';
-import { $$, Quickview, Component, get } from 'coveo-search-ui';
-import { afterQuerySuccess, getRoot, testResultElement, afterDelay } from './Testing';
+import { $$, Quickview, Component, get, Dom } from 'coveo-search-ui';
+import { afterQuerySuccess, getRoot, testResultElement } from './Testing';
+
+function nodeListToArray<T extends Node>(nodeList: NodeListOf<T>) {
+  const result: T[] = [];
+  for (let i = 0; i < nodeList.length; i++) {
+    result.push(nodeList.item(i));
+  }
+  return result;
+}
+
```

Comment:
Could simply use `Dom.nodeListToArray` instead of rewriting it ?

Reply:
Woops, I wasn't aware this function existed. I'll use it instead.